src/main/java/com/pinterest/secor/timestamp/Kafka8MessageTimestamp.java

```diff
@@ -7,11 +7,11 @@
 
     @Override
     public Long getTimestamp(MessageAndMetadata<byte[], byte[]> kafkaMessage) {
-        return null;
+        return 0l;
     }
 
     @Override
     public Long getTimestamp(MessageAndOffset messageAndOffset) {
-        return null;
+        return 0l;
```

Comment:
Looks like both k8 and k10 will return a valid long, should the method now return a small long instead of big Long?